system/jlib/jsocket.cpp

```diff
@@ -1531,7 +1546,17 @@ void CSocket::logPollError(unsigned revents, const char *rwstr)
     if (revents & POLLERR)
     {
         char lname[256];
-        int lport = name(lname, sizeof(lname));
+        int lport;
+        try
+        {
+            lport = name(lname, sizeof(lname));
+        }
+        catch (IException *e)
+        {
+            strcpy(lname, "(NULL)");
+            lport = 1;
+            e->Release();
+        }
```

Comment:
what about other places that call name() , e.g. logConnectionInfo

Should name() always catch and behave like this?